lms/envs/common.py

```diff
@@ -368,7 +368,7 @@
     # This will eventually default to True and may be
     # removed since all installs should have the separate
     # extended history table.
-    'ENABLE_CSMH_EXTENDED': False,
```

Comment:
What is the impact of this?  Also can you make sure that we document the impact in the Juniper release notes

Reply:
It is defaulted to true currently in the defaults in configuration. 

In some environments it causes the CSMExtended application to be included.

I'll mention it in the release notes, but basically it installs that django application, and thus its tables, and starts writing history there, it doesn't migrate the old history or anything.

But it's default on for configuration atm.